### PR TITLE
Define our own prow tide for tekton instead of using openshift

### PR DESCRIPTION
**Pull request by chmouel**
We were using openshift tide rules for tekton let's change that.

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -1115,6 +1110,22 @@ tide:
     repos:
     - redhat-operator-ecosystem/playground
     - redhat-operator-ecosystem/release
+  - labels:
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml]: Needs to be `bugzilla/invalid-bug` in `missingLabels` actually - you do not want to block on *not* having a valid bug.

Comment (chmouel) [reply core-services/prow/02_config/_config.yaml]: ah I got it right at first and thoughts differently, thanks!

File: core-services/prow/02_config/_config.yaml [hunk 2]
```diff
@@ -1115,6 +1110,22 @@ tide:
...
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/blocked-paths
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml]: Trailing whitespace :)

Comment (chmouel) [reply core-services/prow/02_config/_config.yaml]: yep just fixed my emacs :) thanks!

**Comment by petr-muller** [comment]
LOL we need it even here

`Not mergeable. Needs bugzilla/valid-bug label. `

**Comment by chmouel** [comment]
@petr-muller that's on missingLabels right?

**Comment by petr-muller** [comment]
@chmouel Sorry for the confusion. The PR was fine *before*, without the mention of `bugzilla/valid-bug`. What I meant by the comment was that the current configuration wants `bugzilla/valid-bug` even on **this** PR (https://github.com/openshift/release/pull/6918) which is the same type of problem like we're attemptiing to solve here for you.

Please reset back to bc14228, we'll fix openshift/release in the meantime and this PR willl eventually merge.

**Comment by chmouel** [comment]
@petr-muller make sense, thank you!

**Pull request merged**